lighthouse-core/lib/manifest-parser.js

```diff
@@ -242,6 +242,19 @@ function parseIcon(raw, manifestUrl) {
 
   const type = parseString(raw.type, true);
 
+  const purpose = {
+    raw: raw.purpose,
+    /** @type {string[]} */
+    value: [],
+    /** @type {string|undefined} */
+    warning: undefined,
+  };
+  if (raw.purpose !== undefined) {
+    purpose.value = purpose.value.concat(raw.purpose.split(' '));
```

Comment:
```suggestion
    purpose.value = raw.purpose.split(/\s+/).map(purpose => purpose.toLowerCase());
```

Reply:
just from the [ascii whitespace](https://infra.spec.whatwg.org/#ascii-whitespace) comment and "Set canonicalKeyword to ascii lowercased keyword" in the spec :)